src/Symfony/Component/Mime/RawMessage.php

```diff
@@ -47,6 +47,22 @@ public function toIterable(): iterable
             return;
         }
 
+        if ($this->message instanceof \Generator) {
+            if (!$this->message->valid()) {
+                trigger_deprecation('symfony/mime', '6.4', 'Sending an email with a closed generator is deprecated and will throw in 7.0.');
```

Comment:
This deprecation won't be effective: on the first iteration, it is not closed yet (otherwise we already get an exception anyway) and for next iterations, we will have the string assigned on line 61, not the generator.

Reply:
Oops :) Good catch, should be better now.